lxml_html_clean/clean.py

```diff
@@ -90,15 +90,19 @@ def _has_javascript_scheme(s):
 # - 0A - Line Feed
 # - 0B - Vertical tab
 # - 0D - Carriage Return
-_ascii_control_characters = re.compile(r"[\x00-\x08\x0C\x0E-\x1F\x7F]")
+_ascii_control_characters_str = re.compile(r"[\x00-\x08\x0C\x0E-\x1F\x7F]")
+_ascii_control_characters_bytes = re.compile(b"[\x00-\x08\x0C\x0E-\x1F\x7F]")
```

Comment:
```suggestion
_ascii_control_characters_bytes = re.compile(br"[\x00-\x08\x0C\x0E-\x1F\x7F]")
```

Reply:
I am not very excited about the duplicity but OTOH this is the simplest way to do it.

Perhpas add a test that asserts `_ascii_control_characters_str.pattern.encode("ascii") == _ascii_control_characters_bytes.pattern`...

Wait they are not the same. I wonder if the added `r` is needed or not (see suggestion). This confused me. How does the tested case even work? :scream:

Reply:
https://docs.python.org/3/library/re.html#index-37

> Most of the [escape sequences](https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences) supported by Python string literals are also accepted by the regular expression parser:
> 
>     \a      \b      \f      \n
>     \N      \r      \t      \u
>     \U      \v      \x

That explains why this works with or without the `r` prefix.

We talked about this with @frenzymadness and we agreed not to use the r prefix here.